Treat empty :0.0 identically to unset :0.0 when trying to find a PA server. (Closes #87)

From: lennart <lennart@fefdeb5f-60dc-0310-8127-8f9354f1896f>

git-svn-id: svn://svn.0pointer.net/pulseaudio/trunk@1443 fefdeb5f-60dc-0310-8127-8f9354f1896f
---

 src/pulse/client-conf-x11.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/pulse/client-conf-x11.c b/src/pulse/client-conf-x11.c
index 8cedc48..ff145af 100644
--- a/src/pulse/client-conf-x11.c
+++ b/src/pulse/client-conf-x11.c
@@ -42,7 +42,7 @@ int pa_client_conf_from_x11(pa_client_conf *c, const char *dname) {
     int ret = -1;
     char t[1024];
 
-    if (!dname && !getenv("DISPLAY"))
+    if (!dname && (!(dname = getenv("DISPLAY")) || *dname == '\0'))
         goto finish;
     
     if (!(d = XOpenDisplay(dname))) {
